### ISSUE TITLE
Error: Path creation failed: Error: Failed to create /dist

### ISSUE THREAD

**Issue by XescuGC** [chuckmo/grunt-ssh#101]
I'm trying to upload a hole directory to the server using the following configuration:

```
module.exports = {
  deploy: {
    files: {
      './': 'dist/**'
    },
    options: {
      createDirectories: true,
      showProgress: true
    }
  }
}
```

And the output with `-v` is the following:

```
Connection :: connect
Checking existence of path dist/css
Creating /dist
>> Error: Path creation failed: Error: Failed to create /dist
Connection :: end
Copied
Connection :: close
SFTP :: session end
SFTP :: session close
```

In my `sftp` configuration, if I change the `dist/**` to `package.json` it will work, and if i enter the server via sftp i can create new directories. I don't know exactly what's wrong with my configuration, it's exactly the same as the documentation :cry:

**Comment by XescuGC** [comment]
I've solved the problem. In my case when the `grunt-ssh` connects via `sftp` to the server and tries to create the `dist/` folder, it tries to create it in the `/` not in the `/home/deploy`, so it didn't have permission to create it, i've had to add the `path` to the home.
In the documentation it says that: `The path on the remote server. Defaults to home.`, not in my case. Is this a bug? or it's the default behavior of the lib. I will make a PR to add more information to the `Error` and if the documentation is wrong i can update it, and if it's a bug I can try to solve it :smile:

**Comment by XescuGC** [comment]
Closed via #104 and #103
